### PR TITLE
NXP-25094: keep the Download as zip notification displayed

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

File: elements/nuxeo-app.html [hunk 1]
```diff
@@ -1067,7 +1067,8 @@
         var options = e.detail;
         if (options.close) {
           this.$.toast.close();
-        } else if (options.message) {
+        }
+        if (options.message) {
           this.$.toast.text = options.message;
           this.$.toast.duration = options.duration ? options.duration : 3000;
```
Comment (Gabez0r) [inline elements/nuxeo-app.html]: According to the documentation of `paper-toast`, both 0 or a negative number should be used to make it persists, but our check won't work for zero. Perhaps we should do something like `options.duration !== undefined ? options.duration : 3000;` instead?

**Comment by Gabez0r** [comment]
👍

File: elements/nuxeo-document-bulk-actions/nuxeo-download-documents-button.html [hunk 2]
```diff
@@ -106,7 +106,10 @@
               this.fire('notify', {"message": this.i18n('bulkDownload.empty')});
               return;
             }
-            this.fire('notify', {"message": this.i18n('bulkDownload.preparing')});
+            this.fire('notify', {
+              "message": this.i18n('bulkDownload.preparing'),
+              "duration": -1
```
[the remark this answers is not in the document]
Comment (afjusto) [reply elements/nuxeo-document-bulk-actions/nuxeo-download-documents-button.html]: you don't even need the quotes actually

**Comment by afjusto** [comment]
👍

**Pull request merged**